nni/algorithms/compression/pytorch/pruning/iterative_pruner.py

```diff
@@ -51,10 +51,13 @@ def __init__(self, model, config_list, optimizer=None, pruning_algorithm='slim',
         dummy_input: torch.Tensor
             The dummy input to analyze the topology constraints. Note that,
             the dummy_input should on the same device with the model.
+        global_sort: bool
+            If prune the model in a global-sort way.
+            Only support TaylorFOWeightFilterPruner currently.
```

Comment:
if only `TaylorFOWeightFilterPruner` supports `global_sort`, suggest to put this argument in `TaylorFOWeightFilterPruner`.

Reply:
Done.